src/client/signatures.rs

```diff
@@ -112,9 +121,11 @@ impl Verification for DefaultVerifier {
         let pk: EcdsaPublicKey<NistP384> = EcdsaPublicKey::from_bytes(&public_key_bytes)?;
 
         // Instantiate signature
-        let b64_signature = collection.metadata["signature"]["signature"]
-            .as_str()
-            .unwrap_or("");
+        let b64_signature = match collection.metadata["signature"]["signature"].as_str() {
```

Comment:
Ditto -- `.or("")`.

Reply:
I could use ```.or("")``` but I want to unwrap the value inside the option and ```.or("")``` will still yield an Option